k8s/cftptl/common/slack-help-bot/slack-help-bot.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: slack-help-bot
+  namespace: admin
+  annotations:
+    fluxcd.io/automated: "true"
+    fluxcd.io/tag: glob:*
+spec:
+  releaseName: slack-help-bot
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: nodejs
+    version: 2.3.4
+  values:
+    replicas: 2
+    image: hmctspublic.azurecr.io/rpe/slack-help-bot:prod-7c61d17
+    environment:
+      SLACK_REPORT_CHANNEL: slack-bot-test
+      SLACK_REPORT_CHANNEL_ID: C01LQE3E69W
+      JIRA_START_TRANSITION_ID: 21
+      JIRA_DONE_TRANSITION_ID: 41
+      JIRA_PROJECT: SBOX
+      JIRA_ISSUE_TYPE_ID: 10900
+    aadIdentityName: slack-help-bot
+    keyVaults:
+      cftptl-intsvc:
```

Comment:
i don't think this will work, have you tested it? it will automatically add an environment on as a suffix to the keyvault, you probably need to exclude it

Reply:
it might also expect globals/ tenantid. why not move it to kustomize, so they just come in automatically.

Reply:
Sorted

Reply:
u will need https://github.com/hmcts/cnp-flux-config/blob/master/k8s/aat/aat-helmrelease.yaml#L11-L14